shim/shim.c

```diff
@@ -432,6 +446,7 @@ handle_proxy_output(struct cc_shim *shim)
 		code = *(buf + STREAM_HEADER_SIZE); 	// hyperstart has sent the exit status
 		shim_debug("Exit status for container: %d\n", code);
 		free(buf);
+		restoreTerminal();
```

Comment:
Note: this is only an issue when the console is specified by hand in the standalone mode. Docker allocates a new pty in the normal case and it doesn't matter if we leave that one in raw mode.

```
docker kill --signal SIGSEGV $container
```

Shouldn't leave the terminal in raw mode then.